src/pages/Account/ProfileQuestions.tsx

```diff
@@ -125,9 +125,8 @@ export const ProfileQuestions: React.FC = () => {
                 <strong>{question.text}</strong>
                 <textarea
                   className="input-block input-centered"
-                  name={question.name}
                   placeholder={question.text}
-                  ref={register()}
+                  // {...register(question.name)}
```

Comment:
So, keep or delete? A comment like this with no `@debt` explanation doesn't help much.

Reply:
restoredd [9241103](https://github.com/sparkletown/sparkle/pull/2847/commits/9241103be3000f67954e979bcf0e51e9f9b27565)